frontend/packages/component-library/src/list/expandedList/stories/ExpnadedList.story.tsx

```diff
@@ -0,0 +1,83 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import ExpandedList, {ExpandedListProps} from '..';
+
+type Story = StoryObj<typeof ExpandedList>;
+
+export default {
+  title: 'DesignSystem/List/ExpandedList',
+  component: ExpandedList,
```

Comment:
Nit: there are two patterns being applied in Storybook for naming components with 2+ words, camel case and title case. Can we start norming around one so Storybook urls are consistent with multi-word components? Either works for me, but do y'all have a preference cc @levadadenys?

.../?path=/docs/designsystem-**formfieldwrapper**--docs
vs
.../?path=/docs/designsystem-**radio-button**--docs

<img width="283" alt="Screenshot 2025-03-19 at 9 22 25 AM" src="https://github.com/user-attachments/assets/3f8335c8-b823-444f-9999-86712980b0fb" />